File: algorithms/searching/linear_search.py
# Linear search on a unordered array
def linear_search(arr, value):
    for i in range(len(arr)):
        if arr[i] == value:
            return i
    return -1

# Linear search on an ordered array
def linear_search(arr, value):
    for i in range(len(arr)):
        if arr[i] == value:
            return i
        if arr[i] > value:
            break
    return -1

File: algorithms/searching/linear_search/linear_search.js
// On an unordered array
const linearSearch = function(arr, value) {
    for(let i=0; i < arr.length; i++) {
        if(arr[i] == value){
            return i;
        }
    }
    return -1;
}

// On an ordered array
const linearSearchOnOrderedArr = function(arr, value) {
    for (let i=0; i < arr.length; i++) {
        if (arr[i] == value) {
            return i;
        }
        else if (arr[i] > value) {
            break;
        }
    }
    return -1;
}


// Export functions for use in other files
module.exports = {linearSearch, linearSearchOnOrderedArr}
